doc/release/index.rst

```diff
@@ -61,6 +61,12 @@ For information about earlier versions, see :doc:`eol_versions`.
             -   End of support
             -   Versions
 
+        *   -   :doc:`3.3 </release/3.3.0>`
+            -   **November 29, 2024**
+            -   **November 29, 2026**
+            -   **Not planned yet**
+            -   | :tarantool-release:`3.3.0`
+
```

Comment:
This doesn't concern this particular change, but I doubt we'll provide updates and fixes for 3.1 and 3.0 until April 2026/December 2025. We don't do that already.

Am I misunderstanding something?
CC @Totktonada

Reply:
The object of support is the 3.x series at whole. I guess it is what actually meant here, but the representation is confusing. (Likely it is not a problem of this particular 'what's new' article, but a structure of our release informational docs at all.)

Reply:
OK, thanks for spotting, we'll recheck this representation with PMs later to avoid blocking this PR.

Reply:
Sure.